cmd/cli/plugin/package/package_available_get_test.go

```diff
@@ -0,0 +1,159 @@
+        - --provider=rfc2136
+        - --rfc2136-host=100.69.97.77
+        - --rfc2136-port=53
+        - --rfc2136-zone=k8s.example.org
+        - --rfc2136-tsig-secret=MTlQs3NNU=
+        - --rfc2136-tsig-secret-alg=hmac-sha256
+        - --rfc2136-tsig-keyname=externaldns-key
+        - --rfc2136-tsig-axfr
+      env: []
+      securityContext: {}
+      volumeMounts: []
+      volumes: []
+properties:
+  namespace:
+    type: string
+    description: The namespace in which to deploy ExternalDNS.
+    default: external-dns
+    examples:
+      - external-dns
+  deployment:
+    type: object
+    description: Deployment related configuration
+    properties:
+      args:
+        type: array
+        description: |
+          List of arguments passed via command-line to external-dns.  For
+          more guidance on configuration options for your desired DNS
+          provider, consult the ExternalDNS docs at
+          https://github.com/kubernetes-sigs/external-dns#running-externaldns
+        items:
+          type: string
+      env:
+        type: array
+        description: "List of environment variables to set in the external-dns container."
+        items:
+          $ref: "#/definitions/io.k8s.api.core.v1.EnvVar"
+      securityContext:
+        description: "SecurityContext defines the security options the external-dns container should be run with. More info: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/"
+        $ref: "#/definitions/io.k8s.api.core.v1.SecurityContext"
+      volumeMounts:
+        type: array
+        description: "Pod volumes to mount into the external-dns container's filesystem."
+        items:
+          $ref: "#/definitions/io.k8s.api.core.v1.VolumeMount"
+      volumes:
+        type: array
+        description: "List of volumes that can be mounted by containers belonging to the external-dns pod. More info: https://kubernetes.io/docs/concepts/storage/volumes"
+        items:
+          $ref: "#/definitions/io.k8s.api.core.v1.Volume"
+`
+
+var _ = Describe("PackageAvailableGet", func() {
+	var (
+		kappClient   *fakes.KappClient
+		pkgNamespace = "default"
+		pkgName      = "external-dns.tanzu.vmware.com"
+		pkgVersion   = "1.2.1+vmware.1-tkg.2-zshippable"
+	)
+
+	Context("getValuesSchemaForPackage()", func() {
+		BeforeEach(func() {
+			kappClient = &fakes.KappClient{}
+		})
+
+		It("should return error if version is not specified", func() {
+			var buf bytes.Buffer
+			err := getValuesSchemaForPackage(pkgNamespace, pkgName, "", kappClient, &buf)
+			Expect(err).To(HaveOccurred())
+		})
+
+		It("should return error if package does not exist", func() {
+			kappClient.GetPackageReturns(nil,
+				apierrors.NewNotFound(kapppkg.Resource("package"), "dummyPackage"))
+
+			var buf bytes.Buffer
+			err := getValuesSchemaForPackage(pkgNamespace, pkgName, pkgVersion, kappClient, &buf)
+			Expect(err).To(HaveOccurred())
+		})
+
+		It("should return error if schema is empty", func() {
```

Comment:
@danniel1205  thanks for adding the unit test, I have an open PR which not return error if values schema is empty in this case, instead it will return warning msg:
https://github.com/vmware-tanzu/tanzu-framework/pull/932
maybe we can merge these 2 PR together and change this unit test case as well?

Reply:
Adding hold label to this PR. Let's get #932 merged first.